src/events/http.js

```diff
@@ -392,4 +412,10 @@ module.exports = {
             WreckWrapper
         );
     },
+    isURLIgnoredByUser,
+    resolveHttpPromise,
+    USER_AGENTS_BLACKLIST,
+    URL_BLACKLIST,
+    generateEpsagonTraceId,
+    updateAPIGateway,
```

Comment:
move to a common location? not a must let's talk about this

Reply:
👍